nefertari/resource.py

```diff
@@ -4,9 +4,19 @@
 log = logging.getLogger(__name__)
 
 
-ACTIONS = ['index', 'show', 'create', 'update',
-           'delete', 'update_many', 'delete_many',
-           'replace']
+# All actions names(view method names) supported by nefertari
+ACTIONS = [
+    'index',                # Collection GET
+    'create',               # Collection POST
+    'update_many',          # Collection PATCH/PUT
+    'delete_many',          # Collection DELETE
+    'collection_options',   # Collection OPTIONS
+    'show',                 # Item GET
+    'update',               # Item PATCH
+    'replace',              # Item PUT
+    'delete',               # Item DELETE
+    'item_options',         # Item OPTIONS
+]
```

Comment:
Not sure why different permissions are defined for collections and item resources. Why not use the same permissions on both? You can set permissions differently on collection and item resources. Also it probably doesn't make sense to set both say "collection_options" and "item_options" on the same resource.

In my experience one of the most important things to do when setting up an authorization system to to make it understandable and audit-able. Reducing the number of permissions can help.